src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb

```diff
@@ -161,7 +161,11 @@ def unmap(one_vm, disk, directory)
         device = ''
 
         real_path = directory
-        real_path = File.realpath(directory) if File.symlink?(one_vm.sysds_path)
+
+        is_rootfs = real_path.match?(%r{containers(/)one-(\d+)(/)rootfs})
```

Comment:
is it safer to use the same check as in https://github.com/OpenNebula/one/blob/c933c8a514af2b065961844e9e232f69e6dfcb57/src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb#L381?